### DISCUSSION TITLE
[Feature] 终端连接时间支持自定义

### DISCUSSION THREAD

**Discussion by JAINKRE** [1Panel-dev/1Panel#7149] [Q&A]
### 1Panel 版本

v1.10.21-lts

### 请描述您的需求或者改进建议

现在终端长时间连接会自动掉线：

![image](https://github.com/user-attachments/assets/1fe2dcda-426e-4d75-992b-4315bfe808e3)

希望可以自定义超时时间。

**Comment by JAINKRE** [comment]
而且发现，终端超时之后，刷新，面板登录信息也没了，需要重新登录

![image](https://github.com/user-attachments/assets/7aac8eab-f8d9-40c9-8d33-a45156964b92)

**Comment by wanghe-fit2cloud** [comment]
1、`主机` - `终端` 目前没有固定超时时间，但是有心跳保活（十秒发送一次心跳）。
2、`面板登录信息也没了，需要重新登录`，这里可能是因为在 `面板设置` 页面有设置超时时间导致的。

**Reply by JAINKRE** [reply]
这个问题今天又出现了，同样的问题：
> 正在使用的终端突然显示超时，然后面板刷新需要重新登录

日志如下：

![image](https://github.com/user-attachments/assets/67927766-8273-4a66-921e-e581f5cbd678)
